external/webserver/http_client.c

```diff
@@ -633,14 +670,19 @@ int http_recv_and_handle_request(struct http_client_t *client, struct http_keyva
 	conn_type = http_keyvalue_list_find(request_params, "Connection");
 	if (!strncasecmp(conn_type, "Keep-Alive", strlen("Keep-Alive")+1)) {
 		client->keep_alive = 1;
-		struct timeval tv;
-		tv.tv_sec = HTTP_CONF_KEEP_ALIVE_MSEC / 1000;
-		tv.tv_usec = (HTTP_CONF_KEEP_ALIVE_MSEC % 1000) * 1000;
-		HTTP_LOGD("Keep-alive case, change timeout to (%u.%d)sec\n", tv.tv_sec, tv.tv_usec);
-		if (setsockopt(client->client_fd, SOL_SOCKET, SO_RCVTIMEO, (struct timeval *)&tv, sizeof(struct timeval)) < 0) {
-			HTTP_LOGE("Error: Fail to setsockopt\n");
-		} else {
-			HTTP_LOGD("Timeout modified done\n");
+
+		if(client->keep_alive_timeout && client->keep_alive_header_flag == 0) {
```

Comment:
Let's keep the same policy for coding rules.
`if(` or `if (`
We don't follow TizenRT coding rules for external. But we recommend keeping the same policy in the file.

Reply:
done